src/methods/reproject.jl

```diff
@@ -2,15 +2,16 @@
 """
     reproject(obj; crs)
 
-Reproject the lookups of `obj` to a different crs. 
+Reproject the lookups (axes) of `obj` to a different crs. 
```

Comment:
We try to maintain the distinction between axes and lookups as different things

Reply:
I see what you're saying, the problem is that even people who've used rasters for a while have no clue what "lookups" are.  There has to be some frame of reference.  "Dimension values" is also super unclear - what does that mean?

Maybe we can rewrite the parentheses to be more clear?  Would "axis values" work there?

Reply:
FWIW, this is meant to be for people who have no clue what cellarea is or why you might want it.  So you have to go from the ground up, more or less.

Reply:
I get it, but we just can't use a common base function name that also works on a Raster to explain a DD function that's really a different thing.

It's clearly difficult to find words that describe lookups without semantic overloads (see AxisKeys.jl... keys are a different base method again) but it's important that we try. 

I think axis values is helpful, but maybe values is also a pretty empty word. Maybe this needs to be systematic from DD up, we could workshop the language over there.

Reply:
Maybe we could say "x and y values" for now?

Reply:
Lookup values ?

Reply:
Then we're back to the problem of "what is a lookup" again?

Reply:
Maybe we could say "location information".  But I would prefer to just use lookups here and have a glossary where we would explain these differences, because otherwise we would have to duplicate this info everywhere we use lookups.

Reply:
I would argue that we _should_ duplicate this information everywhere we use lookups, at least in top-level, user accessible functions.  I don't want to overestimate the curiosity of a new user v/s their unwillingness to click a link.

Reply:
That's not to say that we shouldn't have a glossary with a more detailed explanation, but there should be something like:

[lookup values](link to DD lookup docs) (the positions of the cells in each dimension)

or something

Reply:
Actually "lookup values (indicating the positions of each cell)" sounds pretty descriptive

Reply:
Actually "lookup values (indicating the positions of each cell)" sounds pretty descriptive